addons/revolutionaries/CfgVehicles_O_Russian.hpp

```diff
@@ -143,6 +143,103 @@ class TACU_Revolutionaries_U_O_Russian_Autorifleman: TACU_Revolutionaries_U_O_Ru
     editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
 };
 
+class TACU_Revolutionaries_U_O_Russian_FigureHeadUA: TACU_Revolutionaries_U_O_Russian_Rifleman01 {
+    displayName = "Figure Head (Unarmed)";
+    uniformClass = "CUP_U_B_USMC_FROG3_WMARPAT";
+    linkedItems[] = {DEFAULT_ITEMS, "CUP_H_USMC_CAP_WDL"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "CUP_H_USMC_CAP_WDL"};
+    backpack = "";
+    weapons[] = {"Throw", "Put"};
+    respawnWeapons[] = {"Throw", "Put"};
+    magazines[] = {""};
+    respawnMagazines[] = {""};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
```

Comment:
```suggestion
    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
    headgearList[] = {
        "CUP_H_USMC_CAP_WDL", 1
    };
```
You also don't have a balaclava under linkedItems and respawnLinkedItems.
This class inherits from the Rifleman01, which means it'll inherit the CBA_facewearList entry as well, giving the unit a balaclava.
If you don't want the unit to have facewear, then add this after headgearList:
```
CBA_facewearList[] = {
    "", 1
};
```

If you _do_ want unit to have a balaclava, then add the item to linkedItems as well.

Reply:
I do still want most of the Units to have Balaclava so will add the item to those and fix